src/App.js

```diff
@@ -1,19 +1,32 @@
 import React, { Component } from "react";
-
+import TouristInfoCards from "./TouristInfoCards";
 import Bookings from "./Bookings";
 import Restaurant from "./Restaurant";
 import "./App.css";
-
+import Heading from "./Heading";
+import ReactDOM from "react-dom";
```

Comment:
The `ReactDOM` import is not necessary. If you look inside `src/index.js` you'll see that we already have set up the `ReactDOM` import & call for you. You don't need to worry about it.

In my examples that we looked at in class it is needed because the entire application is in 1 file

Reply:
done